example/driver/simpledriver.go

```diff
@@ -25,6 +30,38 @@ type SimpleDriver struct {
 	switchButton bool
 }
 
+func getImageBytes(imgFile string, buf *bytes.Buffer) error {
+	// Read existing image from file
+	img, err := os.Open(imgFile)
+	if err != nil {
+		return err
+	}
+	defer img.Close()
+
+	// TODO: Attach MediaType property, determine if decoding
```

Comment:
It may save cycles to just add it to this PR then.
My intent was to foster a conversation via code/PR (treating master as a typical feature branch) since that seemed to be the operating model. But I'll adjust.

Reply:
No need to add, just please add an explanation to the PR description. The idea is to provide as much context and information in order facilitate review of the PR. @anonymouse64 writes some really good PR descriptions for snap changes that many times include how to test the PR.

Reply:
:thumbsup: Got it. Okay will address this in a separate PR and updated this PR's description.